Core/include/Acts/SpacePointFormation/SpacePointBuilderOptions.hpp

```diff
@@ -8,12 +8,40 @@
 
 #pragma once
 
+#include "Acts/EventData/SourceLink.hpp"
+
 namespace Acts {
 
 struct SpacePointBuilderOptions {
+  // ends of strip pairs
   std::pair<const std::pair<Vector3, Vector3>,
             const std::pair<Vector3, Vector3>>
       stripEndsPair;
+  // accessor of local position and covariance from soruce link
+  std::function<std::pair<const BoundVector, const BoundSymMatrix>(SourceLink)>
+      paramCovAccessor;
+  /// vertex position
+  Vector3 vertex = {0., 0., 0.};
+  /// Allowed increase of strip length
+  double stripLengthTolerance = 0.01;
+  /// Allowed increase of strip length wrt gaps between strips
+  double stripLengthGapTolerance = 0.01;
+  SpacePointBuilderOptions() = default;
+};
+
+struct StripPairOptions {
+  // accessor of local position and covariance from soruce link
+  std::function<std::pair<const BoundVector, const BoundSymMatrix>(SourceLink)>
+      paramCovAccessor;
+  /// vertex position
+  Vector3 vertex = {0., 0., 0.};
+  /// Accepted squared difference in theta for two clusters
+  double diffTheta2 = 1.;
+  /// Accepted squared difference in phi for two clusters
+  double diffPhi2 = 1.;
+  /// Accepted distance between two clusters
+  double diffDist = 100. * UnitConstants::mm;
+  StripPairOptions() = default;
```

Comment:
It depends on users. In athena, we need to set the values in the option here for each SP candidate. The other values are in the config. The StripPairOptions for the strip pair making is actually not used anywhere yet. It could be used in the Example when the strip digitization becomes available.